cmd/server.go

```diff
@@ -259,6 +260,9 @@ var stringFlags = map[string]stringFlag{
 			"This means that an attacker could spoof calls to Atlantis and cause it to perform malicious actions. " +
 			"Should be specified via the ATLANTIS_GITLAB_WEBHOOK_SECRET environment variable.",
 	},
+	APISecretFlag: {
```

Comment:
This is fine with me, but my use of this feature would probably restrict the plan and apply via HTTP rules at the ingress layer.

Reply:
Thanks for the feedback!